.vib/redis-sentinel/goss/redis-sentinel.yaml

```diff
@@ -0,0 +1,17 @@
+file:
+  /opt/bitnami/redis-sentinel/etc/sentinel.conf:
+    exists: true
+    contains:
+      - port 26379
+      - bind 0.0.0.0
+      - /pidfile.*/opt/bitnami/redis-sentinel/tmp/redis-sentinel.pid/
+      - daemonize no
+      - logfile ""
+command:
+  check-redis-server:
+    exec: timeout --preserve-status 5 redis-server /opt/bitnami/redis-sentinel/etc/sentinel.conf --sentinel
+    exit-status: 0
+    timeout: 10000
+    stdout:
+      - Configuration loaded
+      - /monitor.*quorum/
```

Comment:
I think you should also check for SSL just like you do in `redis-cluster` or `redis`.

https://github.com/bitnami/containers/blob/0e2db42bcebb6c2925092ac5a53d7653025ddbf5/.vib/redis-cluster/goss/redis-cluster.yaml#L18

https://github.com/bitnami/containers/blob/0e2db42bcebb6c2925092ac5a53d7653025ddbf5/.vib/redis-cluster/goss/redis-cluster.yaml#L24

Reply:
Thanks for catching this, updated!